sherpa/utils/__init__.py

```diff
@@ -2113,6 +2116,10 @@ def histogram2d(x, y, x_grid, y_grid):
 
     See the `numpy.histogram2d` routine for a version with more options.
 
+    .. versionchanged:: 4.15.0
+       The x_grid and x_grid arguments are no-longer changed (sorted)
```

Comment:
```suggestion
       The x_grid and y_grid arguments are no-longer changed (sorted)
```

Reply:
Ta. One day I will create a PR without a typo or thinko. One day. But not today.